src/nvim/quickfix.c

```diff
@@ -4109,6 +4113,34 @@ enum {
   QF_GETLIST_ALL = 0xFF
 };
 
+// Parse text from 'di' and return the quickfix list items
+static int qf_get_list_from_text(dictitem_T *di, dict_T *retdict)
+{
+  int status = FAIL;
+
+  // Only string and list values are supported
+  if ((di->di_tv.v_type == VAR_STRING
+       && di->di_tv.vval.v_string != NULL)
+      || (di->di_tv.v_type == VAR_LIST
+          && di->di_tv.vval.v_list != NULL)) {
+    qf_info_T *qi = xmalloc(sizeof(*qi));
+    memset(qi, 0, sizeof(*qi));
```

Comment:
Why doesn't Vim use `calloc` here? Is it buggy or unavailable on some platforms?

Reply:
@janlazo we use `xcalloc` in some cases, but it probably doesn't matter since the compiler optimizes it anyway.